docs/book/controllers.md

```diff
@@ -17,9 +17,11 @@ class Foo implements DispatchableInterface
 }
 ```
 
-While the pattern is straight-forward, chances are you don't want to implement
-custom dispatch logic for every controller, particularly as it's not unusual or
-uncommon for a single controller to handle several related types of requests.
+This approach allows you to handle only one type of request per class. This can help keep classes small.
+
+If you wish to handle multiple types of requests per class, there are several abstract controllers that you can extend:
```

Comment:
Add an empty line after this, before the bullet items. (This is part of our style guide.)

I'll take care of this and then merge.